.github/workflows/ci.yml

```diff
@@ -489,6 +489,22 @@ jobs:
           env
           sudo -E PATH=$PATH ./script/critest.sh "${{github.workspace}}/report"
 
+      - name: Install tools
+        # The GitHub Actions image already has buildah and podman included. Not the
+        # arm64-8core-32gb images. buildah and podman  is needed to convert the
+        # Kubernetes checkpoint archive to an OCI image.
+        # See contrib/checkpoint/checkpoint-restore-cri-test.sh
+        if: matrix.os == 'arm64-8core-32gb'
```

Comment:
This one has been moved to GHA arm image:

https://github.com/containerd/containerd/pull/11427

Reply:
Changed.